test/infrastructure/docker/docker/machine.go

```diff
@@ -234,16 +234,18 @@ func (m *Machine) ExecBootstrap(ctx context.Context, data string) error {
 	}
 
 	m.log.Info("Running machine bootstrap scripts")
-	var out bytes.Buffer
+	var outErr bytes.Buffer
+	var outStd bytes.Buffer
 	for _, command := range commands {
 		cmd := m.container.Commander.Command(command.Cmd, command.Args...)
-		cmd.SetStderr(&out)
+		cmd.SetStderr(&outErr)
+		cmd.SetStdout(&outStd)
 		if command.Stdin != "" {
 			cmd.SetStdin(strings.NewReader(command.Stdin))
 		}
 		err := cmd.Run(ctx)
 		if err != nil {
-			m.log.Info("Failed running command", "command", command, "stderr", out.String())
+			m.log.Info("Failed running command", "command", command, "stdout", outStd.String(), "stderr", outErr.String(), "cloud init", cloudConfig)
```

Comment:
cloudConfig shows up b64 encoded in the log. Maybe we don't need it though?

Reply:
If we print plain text it gets super long and it gets also un-usable due to escaping ( \n \" added here and there)

Never the less, not matter if we are keeping the encoded bootstrap data, I think it is useful for debugging. 
NB. I don't think it is a problem having CA in CI logs because the cluster are not accessible from the outside (and the logs are published on prow only after the cluster is deleted)